charts/opensearch-dashboards/values.yaml

```diff
@@ -162,6 +162,7 @@ ingress:
   annotations: {}
     # kubernetes.io/ingress.class: nginx
     # kubernetes.io/tls-acme: "true"
+  labels: {}
```

Comment:
Can we add a [documentation](https://github.com/opensearch-project/helm-charts/blob/main/charts/opensearch-dashboards/README.md) for this change so that it is easy for new users?

Reply:
The ingress object is not documented at all currently.
```
ingress | Configurable [ingress][] to expose the OpenSearch service. See [values.yaml][] for an example | see [values.yaml][]
```
It would be a bit weird to just add labels and it seems too broad a change to document all the ingress parameters in this PR.  Are you willing to accept a subsequent PR that documents all ingress parameters in one pass?

Reply:
@jwitko yes a separate PR makes sense. I will create a separate issue for this, if you wish to pick it up it will be great.

Reply:
Created a new issue #405